include/ion/node.h

```diff
@@ -126,6 +127,31 @@ class Node {
     {
     }
 
+    Node(const std::string& id, const std::string& name, const Halide::Target& target, const std::string& graph_id)
+        : impl_(new Impl{id, name, target, graph_id})
+    {
+    }
+
+
+    template<typename... Args>
+    std::vector<Port> create_ports_vector(Args... args) const {
```

Comment:
Can you please clarify necessity of the wrapper function?

Reply:
Sorry for the confusion,  this is unnecessary and I delete it